getting_started/introduction/godot_design_philosophy.rst

```diff
@@ -141,6 +141,14 @@ programmers work on features they may need themselves, so you'll see
 improvements in all corners of the engine at the same time in every
 major release.
 
+.. note::
+
+   Community-driven does not mean that decisions are made by the community as
+   such. The Godot's governance model may be based on the feedback of the
+   community which directly influence, lead, and guide the final decision-making
+   of the core developers in relation to Godot's development and shape the
+   future behind Godot as a project.
```

Comment:
```suggestion
   Community-driven does not mean that all decisions are made by voting. 
   Decisions are made by trusted community members who have shown
   dedication, skill, and leadership within the community. The feedback of 
   the community directly guides final decisions in relation to Godot's 
   development and shapes the future behind Godot as a project.
```

Reply:
I disagree because:

1. Decisions are not made by voting. It's been discussed earlier by other core developers that voting only bring up necessary attention to the core dev team. But voting (or consensus) do influence *final decisions*.
2. The feedback of the community does not directly guides *final decisions*. It's the *process* of decision-making which *may* be influenced by the community. The community feedback *may* be the driving force for a change, but it's not always the case that the fuel for change is purely community-based, there may be other motivations for a change.

I can amend my wording to be slightly more clear on semantical level, but I cannot apply these suggestions for the reasons above.

Reply:
Here's a slightly more concise and improved version of my own suggestion (so I don't forget about it):

```suggestion
   Community-driven does not mean that final decisions are made by the community as
   such. Rather, Godot's development is based on community feedback which directly
   guides the process of decision-making of trusted core developers and maintainers
   that shape the future behind Godot as a project.
```

Of course, maintainers are also part of community, nobody denies that.

1. Avoid the usage of "voting" word, because it's not democracy here, and never was.
2. Not being specific on how trust is earned. I think it's totally subjective. Read reduz opinion on this, for instance: 
   - https://twitter.com/reduzio/status/1376859320743235585

Reply:
At the very least, Clay's version reads better because it doesn't try to be one sentence. The more complex you make sentences, the more dependent parts there are, the harder they are to comprehend, which is against the goal of this PR.

Clay's take has 3 sentences answering individual questions:
1. What "community-driven" isn't.
2. Who makes the final decision.
3. How is community involved then.

Reply:
I agree, the clarity can be improved. But simplification may lead to poor quality of the message as a whole.

If I were to suggest the edited clayjohn version, it would look something like this:

```suggestion
   Community-driven does not mean that final decisions are made by the community
   at large. Decisions are made by community members who have earned trust within
   the community. Yet community feedback directly guides this decision-making
   process in relation to Godot's development and shapes the future behind Godot
   as a project.
```

Which I think may be a good compromise between our versions.

Reply:
I just woke up, so I may not be at 100% brain power, but here's my suggestion. I think it reads a little better? If not, I'll probably revise it later today.

```suggestion
   Community-driven does not mean that final decisions are made by the community
   at large. Rather, decisions are made by trusted community members who have
   shown dedication, skill, and leadership within the community. However,
   community feedback often directly guides this decision-making process, which
   allows the shaping of Godot's future as a project.
```

Note: I use "often" when saying it guides the decision-making process, as there are some cases where community feedback, even positive feedback, may not change the closure of a proposal.

Reply:
*"which allows to shape Godot's future as a truly community-driven project"* may read better as well, and may be worth repeating "community-driven" at the end to emphasize on the importance of the community feedback, or at least makes it more optimistic, IMO. 🙂

Reply:
Hmm. How's this sound, then?

```suggestion
   Community-driven does not mean that final decisions are made by the community
   at large. Rather, decisions are made by trusted community members who have
   shown dedication, skill, and leadership within the community. However,
   community feedback often directly guides this decision-making process, which
   in turn allowing the community to shape Godot's future as a truly
   community-driven project.
```

I use "in turn" here, but if it reads better without it, you could remove it and probably have the same message.

Reply:
I think this is pretty much it! I can commit this, but see https://github.com/godotengine/godot-docs/pull/4799#issuecomment-812492242.

Reply:
I committed all of the above suggestions based on consensus (thanks to LikeLakers2 for summing it up in a final text). Up to further discussion, of course.